### PR TITLE
null is awaited so no need to test for null #2085

### PR DESCRIPTION
**Pull request by bhugot**

### CODE REVIEW

File: src/Paket.Core/ProjectFile.fs [hunk 1]
```diff
@@ -200,17 +200,8 @@ module ProjectFile =
...
-        let namespaceURI = 
-            try
-                project.Document.DocumentElement.NamespaceURI
-            with
-            | _ -> null
-        if String.IsNullOrWhiteSpace namespaceURI then
-            project.Document.CreateElement (name, Constants.ProjectDefaultNameSpace)
-        else
-            project.Document.CreateElement (name, namespaceURI)
+        project.Document.CreateElement (name, project.Document.DocumentElement.NamespaceURI)
```
Comment (forki) [inline src/Paket.Core/ProjectFile.fs]: are you sure this doesn't crash on weird input?

Comment (bhugot) [inline src/Paket.Core/ProjectFile.fs]: It s working on dotnet core csproj like a charm and tests are passing but 2 who where already down

**Comment by forki** [comment]
Can you please still put it under try catch? I have a weird feeling here.

Am 20.12.2016 9:52 nachm. schrieb "Benjamin Hugot" <notifications@github.com
